### DISCUSSION TITLE
Craft Plugin Store: Change default Plugin sort order

### DISCUSSION THREAD

**Discussion by tomkiss** [craftcms/cms#9584] [Ideas] [answered]
I just noticed that the Craft Plugin Store doesn't seem to de-prioritise Abandoned plugins. At least, the default sorting of "popularity" is kind of unhelpful in some circumstances:

![image](https://user-images.githubusercontent.com/183310/125917249-0db72cde-343d-492c-9e76-457ae56a079e.png)

Would be good if the "Abandoned" Plugins were pushed to the back? Even if they're popular?

kthxbai

**Comment by brandonkelly** [comment, marked as answer]
Good call! Just made that change. [Already in effect](https://plugins.craftcms.com/search?q=imager).
